pkg/path/Earthfile

```diff
@@ -0,0 +1,9 @@
+VERSION 0.7
```

Comment:
that's a lot of files for everything in `pkg`.
Could we instead have one file `pkg/Earthfile` to take care of all of it?

Reply:
It turned out to be easier than I thought. Now there is only one Earthfile in pkg